static/src/js/components/Map/LayerSwitcherControl.scss

```diff
@@ -4,23 +4,37 @@
   position: relative;
   border-radius: 2px;
 
-  button {
+  &__button {
     width: 1.75rem;
     height: 1.75rem;
+
+    &--show {
+      .layer-switcher-control--visible & {
+        display: none;
+      }
+    }
+
+    &--hide {
+      display: none !important;
```

Comment:
Might be worth commenting that these are needed to override styles in openlayers.